README.md

```diff
@@ -315,10 +315,18 @@ Developers can compare multiple experiments by specifying and uploading the path
 
 ### Graph
 
-**Graph** enables developers to visualize model structures by only one click. Moreover, **Graph** allows developers to explore model attributes, node information, node input and output. aiding them analyze model structures quickly and understand the direction of data flow easily.
+**Graph** enables developers to visualize model structures by only one click. Moreover, **Graph** allows developers to explore model attributes, node information, node input and output. aiding them analyze model structures quickly and understand the direction of data flow easily. Currently we provide two versions to visualize dynamic and static model graph individually.
```

Comment:
**Graph** enables developers to visualize model structures by only one click. Moreover, **Graph** allows developers to explore model attributes, node information, node input and output, aiding them analyze model structures quickly and understand the direction of data flow easily. Additionally, **Graph** supports the visualization of dynamic and static model graph respectively.

Reply:
Done